src/azure-cli-core/azure/cli/core/tests/test_aaz_field.py

```diff
@@ -59,14 +59,13 @@ def test_aaz_model_and_simple_types(self):
         model_schema.properties.enable = AAZBoolType()
         v.properties.enable = True
         assert not v.properties.enable._is_patch
-        assert v.properties.enable == True
```

Comment:
I feel this line is deliberate as it wants to test the [`__eq__`](https://docs.python.org/3/reference/datamodel.html#object.__eq__) operator, so it should not be removed.

Reply:
I'll reinsert it, with `# noqa` to add support for ruff/flake8.

Reply:
Yes, please unchanged this lines. BTW, please don't change the code here https://github.com/search?q=repo%3AAzure%2Fazure-cli+path%3A%2F%5Esrc%5C%2Fazure-cli-core%5C%2Fazure%5C%2Fcli%5C%2Fcore%5C%2Faaz%5C%2F%2F+%2F%3D%3D+None%2F&type=code

Reply:
it has been moved to line 62. I'll shuffle them around, so the diff is more clear that I only added the `# noqa`-bit.

Reply:
The line has been added back :)